diff --git a/node_modules/@slightlyoff/lite-vimeo/lite-vimeo.js b/node_modules/@slightlyoff/lite-vimeo/lite-vimeo.js
index c003b49..f6ca0e6 100644
--- a/node_modules/@slightlyoff/lite-vimeo/lite-vimeo.js
+++ b/node_modules/@slightlyoff/lite-vimeo/lite-vimeo.js
@@ -142,6 +142,7 @@ let LiteVimeoEmbed = /** @class */ (() => {
           border-radius: 10%;
           transition: all 0.2s cubic-bezier(0, 0, 0.2, 1);
           border: 0;
+          cursor: pointer;
         }
         #frame:hover .lvo-playbtn {
           background-color: rgb(98, 175, 237);
