diff --git a/node_modules/@smui/icon-button/IconButton.svelte b/node_modules/@smui/icon-button/IconButton.svelte
index d5988b8..a1d9ba9 100644
--- a/node_modules/@smui/icon-button/IconButton.svelte
+++ b/node_modules/@smui/icon-button/IconButton.svelte
@@ -14,7 +14,6 @@
       {context === 'snackbar' ? 'mdc-snackbar__dismiss' : ''}
     "
     use:Ripple={{ripple: ripple && !toggle, unbounded: true, color}}
-    aria-hidden="true"
     aria-pressed={pressed}
     {href}
     on:MDCIconButtonToggle:change={handleChange}
@@ -36,7 +35,6 @@
       {context === 'snackbar' ? 'mdc-snackbar__dismiss' : ''}
     "
     use:Ripple={{ripple: ripple && !toggle, unbounded: true, color}}
-    aria-hidden="true"
     aria-pressed={pressed}
     on:MDCIconButtonToggle:change={handleChange}
     {...props}
