diff --git a/node_modules/d3-geo-projection/src/satellite.js b/node_modules/d3-geo-projection/src/satellite.js
index d184386..d6ff552 100644
--- a/node_modules/d3-geo-projection/src/satellite.js
+++ b/node_modules/d3-geo-projection/src/satellite.js
@@ -1,5 +1,5 @@
 import {geoProjectionMutator as projectionMutator} from "d3-geo";
-import {acos, asin, atan2, cos, degrees, radians, sin, sqrt} from "./math.js";
+import {acos, asin, atan2, cos, degrees, epsilon, radians, sin, sqrt} from "./math.js";
 
 function satelliteVerticalRaw(P) {
   function forward(lambda, phi) {
@@ -57,7 +57,8 @@ export default function() {
   // As a multiple of radius.
   p.distance = function(_) {
     if (!arguments.length) return distance;
-    return m(distance = +_, omega);
+    return m(distance = +_, omega)
+      .clipAngle(acos(1 / distance) * degrees - epsilon);
   };
 
   p.tilt = function(_) {
@@ -67,5 +68,5 @@ export default function() {
 
   return p
       .scale(432.147)
-      .clipAngle(acos(1 / distance) * degrees - 1e-6);
+      .clipAngle(acos(1 / distance) * degrees - epsilon);
 }
