diff --git a/LICENSE b/LICENSE
index 561dff22877200931fd57b0fb6f5dda909bb3472..828f791f18d4a6fbe39529348dfb3368674b2a67 100644
--- a/LICENSE
+++ b/LICENSE
@@ -1,6 +1,7 @@
 
 Copyright 2020 Shane Holloway. (ES Module port)
 Copyright 2019 David Bau.
+Copyright (C) 2010 by Johannes Baagøe <baagoe@baagoe.org>
 
 Permission is hereby granted, free of charge, to any person obtaining
 a copy of this software and associated documentation files (the
diff --git a/package.json b/package.json
index e24a5c8be10319e2b52edbcb680a8ef46a6143bc..afea8e6c72f0c782d0ea05ea95127a8e09b1ee58 100644
--- a/package.json
+++ b/package.json
@@ -11,7 +11,8 @@
     "type": "git",
     "url": "git://github.com/shanewholloway/js-esm-seedrandom.git"
   },
-  "author": [
+  "contributors": [
+    "Johannes Baagøe <baagoe@baagoe.org>",
     "David Bau",
     "Shane Holloway"
   ],
