diff --git a/build/quantities.mjs b/build/quantities.mjs
index 1a25dded9c6b6faa01063609d7acd87973133619..a340af405ee2cea481e71c980898a55e0a2803db 100644
--- a/build/quantities.mjs
+++ b/build/quantities.mjs
@@ -338,6 +338,7 @@ var UNITS = {
   /* concentration */
   "<molar>" : [["M","molar"], 1000, "concentration", ["<mole>"], ["<meter>","<meter>","<meter>"]],
   "<wtpercent>"  : [["wt%","wtpercent"], 10, "concentration", ["<kilogram>"], ["<meter>","<meter>","<meter>"]],
+  "<dobson>"  : [["DU","Dobson unit"], 4.4615e-4, "concentration", ["<mole>"], ["<meter>","<meter>"]],
 
   /* activity */
   "<katal>" :  [["kat","katal","Katal"], 1.0, "activity", ["<mole>"], ["<second>"]],