diff --git a/node_modules/ky/index.js b/node_modules/ky/index.js
index 897c9f0..38580fd 100644
--- a/node_modules/ky/index.js
+++ b/node_modules/ky/index.js
@@ -468,7 +468,7 @@ class Ky {
 
 		return new globals.Response(
 			new globals.ReadableStream({
-				start(controller) {
+				async start(controller) {
 					const reader = response.body.getReader();
 
 					if (onDownloadProgress) {
@@ -489,10 +489,10 @@ class Ky {
 						}
 
 						controller.enqueue(value);
-						read();
+						await read();
 					}
 
-					read();
+					await read();
 				}
 			})
 		);
