diff --git a/dist/getPackageResolution.js b/dist/getPackageResolution.js
index daf48d92d70e2eede8e51d3e3a6188be6fd74358..e78503cfd461c7c556f6f0859f62d2404106114e 100644
--- a/dist/getPackageResolution.js
+++ b/dist/getPackageResolution.js
@@ -77,6 +77,9 @@ function getPackageResolution({ packageDetails, packageManager, appPath, }) {
         const lockfile = require(path_1.join(appPath, packageManager === "npm-shrinkwrap"
             ? "npm-shrinkwrap.json"
             : "package-lock.json"));
+        if (lockfile.lockfileVersion === 3) {
+            return lockfile.packages[packageDetails.path].resolved;
+        }
         const lockFileStack = [lockfile];
         for (const name of packageDetails.packageNames.slice(0, -1)) {
             const child = lockFileStack[0].dependencies;