diff --git a/node_modules/rollup-plugin-glslify/index.js b/node_modules/rollup-plugin-glslify/index.js
index 85474ec..75f9c8a 100644
--- a/node_modules/rollup-plugin-glslify/index.js
+++ b/node_modules/rollup-plugin-glslify/index.js
@@ -58,7 +58,12 @@ module.exports = function glslify(userOptions = {}) {
                 options
             );
 
-            code = compile(code, fileOptions);
+            const { source, deps } = compile(code, fileOptions);
+            code = source;
+
+            for (const dep of deps.filter(dep => !dep.entry)) {
+              this.addWatchFile(dep.file);
+            }
 
             if (typeof options.compress === 'function') {
                 code = options.compress(code);
