diff --git a/node_modules/rollup-plugin-license/dist/index.js b/node_modules/rollup-plugin-license/dist/index.js
index 627755f..8521cbc 100644
--- a/node_modules/rollup-plugin-license/dist/index.js
+++ b/node_modules/rollup-plugin-license/dist/index.js
@@ -903,7 +903,7 @@ class LicensePlugin {
           pkg = pkgJson; // Read license file, if it exists.
 
           const licenseFile = glob__default["default"].sync(
-            path__default["default"].join(dir, "LICENSE*")
+            path__default["default"].join(dir, "{LICENSE,license}*")
           )[0];
 
           if (licenseFile) {
