diff --git a/index.js b/index.js
index 85474ec0aeaed7104ccd6b4784541ed7c85220dc..2c1e423d1b155e457d974935f5049fad65165f03 100644
--- a/index.js
+++ b/index.js
@@ -58,7 +58,12 @@ module.exports = function glslify(userOptions = {}) {
                 options
             );
 
-            code = compile(code, fileOptions);
+            const { source, deps } = compile(code, fileOptions);
+            code = source;
+
+            for (const dep of deps.filter(dep => !dep.entry)) {
+              this.addWatchFile(dep.file);
+            }
 
             if (typeof options.compress === 'function') {
                 code = options.compress(code);
