diff --git a/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte b/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
index 1f12c22..49f889a 100755
--- a/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
+++ b/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
@@ -18,6 +18,15 @@
   export let reversed = false;
   export let hoverable = true;
   export let disabled = false;
+  export let value = values[0];
+
+  // keep value and values in sync with each other
+  const updateValues = () => { if (value !== values[0]) values = [value] };
+  const updateValue = () => {
+    if (alignValueToStep(value) !== values[0]) value = values[0];
+  };
+  $: value, updateValues();
+  $: values, updateValue();
 
   // range pips / values props
   export let pips = false;
@@ -592,8 +601,8 @@
   :global(.rangeSlider) {
     position: relative;
     border-radius: 100px;
-    height: 0.5em;
-    margin: 1em;
+    height: 8px;
+    margin: 16px;
     transition: opacity 0.2s ease;
     user-select: none;
   }
@@ -601,31 +610,31 @@
     user-select: none;
   }
   :global(.rangeSlider.pips) {
-    margin-bottom: 1.8em;
+    margin-bottom: 29px;
   }
   :global(.rangeSlider.pip-labels) {
-    margin-bottom: 2.8em;
+    margin-bottom: 45px;
   }
   :global(.rangeSlider.vertical) {
     display: inline-block;
     border-radius: 100px;
-    width: 0.5em;
+    width: 8px;
     min-height: 200px;
   }
   :global(.rangeSlider.vertical.pips) {
-    margin-right: 1.8em;
-    margin-bottom: 1em;
+    margin-right: 29px;
+    margin-bottom: 16px;
   }
   :global(.rangeSlider.vertical.pip-labels) {
-    margin-right: 2.8em;
-    margin-bottom: 1em;
+    margin-right: 45px;
+    margin-bottom: 16px;
   }
   :global(.rangeSlider .rangeHandle) {
     position: absolute;
     display: block;
-    height: 1.4em;
-    width: 1.4em;
-    top: 0.25em;
+    height: 22px;
+    width: 22px;
+    top: 4px;
     bottom: auto;
     transform: translateY(-50%) translateX(-50%);
     z-index: 2;
@@ -634,7 +643,7 @@
     transform: translateY(-50%) translateX(50%);
   }
   :global(.rangeSlider.vertical .rangeHandle) {
-    left: 0.25em;
+    left: 4px;
     top: auto;
     transform: translateY(50%) translateX(-50%);
   }
@@ -647,7 +656,7 @@
     left: 0;
     top: 0;
     display: block;
-    border-radius: 10em;
+    border-radius: 160px;
     height: 100%;
     width: 100%;
     transition: box-shadow 0.2s ease;
@@ -673,7 +682,7 @@
     opacity: 0.4;
   }
   :global(.rangeSlider.range:not(.min):not(.max) .rangeNub) {
-    border-radius: 10em 10em 10em 1.6em;
+    border-radius: 160px 160px 160px 26px;
   }
   :global(.rangeSlider.range .rangeHandle:nth-of-type(1) .rangeNub) {
     transform: rotate(-135deg);
@@ -703,36 +712,35 @@
     display: block;
     position: absolute;
     left: 50%;
-    top: -0.5em;
+    top: -8px;
     transform: translate(-50%, -100%);
-    font-size: 1em;
     text-align: center;
     opacity: 0;
     pointer-events: none;
     white-space: nowrap;
     transition: all 0.2s ease;
-    font-size: 0.9em;
-    padding: 0.2em 0.4em;
-    border-radius: 0.2em;
+    font-size: 0.9rem;
+    padding: 3px 6px;
+    border-radius: 3px;
   }
   :global(.rangeSlider .rangeHandle.active .rangeFloat),
   :global(.rangeSlider.hoverable .rangeHandle:hover .rangeFloat) {
     opacity: 1;
-    top: -0.2em;
+    top: -3px;
     transform: translate(-50%, -100%);
   }
   :global(.rangeSlider .rangeBar) {
     position: absolute;
     display: block;
     transition: background 0.2s ease;
-    border-radius: 1em;
-    height: 0.5em;
+    border-radius: 16px;
+    height: 8px;
     top: 0;
     user-select: none;
     z-index: 1;
   }
   :global(.rangeSlider.vertical .rangeBar) {
-    width: 0.5em;
+    width: 8px;
     height: auto;
   }
   :global(.rangeSlider) {
