diff --git a/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte b/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
index 0307932..b484b58 100755
--- a/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
+++ b/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
@@ -12,6 +12,13 @@
   export let vertical = false;
   export let float = false;
   export let hover = true;
+  export let value = values[0];
+
+  // keep value and values in sync with each other
+  const updateValues = () => { if (value !== values[0]) values = [value] };
+  const updateValue = () => { if (value !== values[0]) value = values[0] };
+  $: value, updateValues();
+  $: values, updateValue();
 
   // range pips / values props
   export let pips = false;
