diff --git a/LICENSE-MPL-2.0 b/LICENSE-MPL-2.0
deleted file mode 100644
index 14e2f777f6c395e7e04ab4aa306bbcc4b0c1120e..0000000000000000000000000000000000000000
diff --git a/src/RangeSlider.svelte b/src/RangeSlider.svelte
index 460bf5c0bb18d349217644b625e4816a0b1ea656..dafbbf129bd599e821f7c3e42e4342fbfc2804af 100644
--- a/src/RangeSlider.svelte
+++ b/src/RangeSlider.svelte
@@ -20,7 +20,16 @@
   export let reversed = false;
   export let hoverable = true;
   export let disabled = false;
+  export let value = values[0];
 
+  // keep value and values in sync with each other
+  const updateValues = () => { if (value !== values[0]) values = [value] };
+  const updateValue = () => {
+    if (alignValueToStep(value) !== values[0]) value = values[0];
+  };
+  $: value, updateValues();
+  $: values, updateValue();
+ 
   // range pips / values props
   export let pips = false;
   export let pipstep = undefined;
@@ -799,6 +808,7 @@
 </style>
 
 <div
+  role="presentation"
   {id}
   bind:this={slider}
   class="rangeSlider"
