diff --git a/LICENSE-MPL-2.0 b/LICENSE-MPL-2.0
deleted file mode 100755
index 14e2f777f6c395e7e04ab4aa306bbcc4b0c1120e..0000000000000000000000000000000000000000
diff --git a/src/RangeSlider.svelte b/src/RangeSlider.svelte
index 60886c96b312e5e2ee83d7ac624a46ac4512f5c5..16cf2bacedd7baeef62fa1044226f2a2bf7d3e50 100755
--- a/src/RangeSlider.svelte
+++ b/src/RangeSlider.svelte
@@ -20,6 +20,15 @@
   export let reversed = false;
   export let hoverable = true;
   export let disabled = false;
+  export let value = values[0];
+
+  // keep value and values in sync with each other
+  const updateValues = () => { if (value !== values[0]) values = [value] };
+  const updateValue = () => {
+    if (alignValueToStep(value) !== values[0]) value = values[0];
+  };
+  $: value, updateValues();
+  $: values, updateValue();
 
   // range pips / values props
   export let pips = false;
@@ -787,6 +796,7 @@
 </style>
 
 <div
+  role="presentation"
   {id}
   bind:this={slider}
   class="rangeSlider"
