diff --git a/node_modules/svelte/compiler.mjs b/node_modules/svelte/compiler.mjs
index b9045c9..0a11f3f 100644
--- a/node_modules/svelte/compiler.mjs
+++ b/node_modules/svelte/compiler.mjs
@@ -32099,9 +32099,6 @@ function process_component_options(component, nodes) {
                         if (tag && !/^[a-zA-Z][a-zA-Z0-9]*-[a-zA-Z0-9-]+$/.test(tag)) {
                             return component.error(attribute, compiler_errors.invalid_tag_property);
                         }
-                        if (tag && !component.compile_options.customElement) {
-                            component.warn(attribute, compiler_warnings.missing_custom_element_compile_options);
-                        }
                         component_options.tag = tag;
                         break;
                     }
diff --git a/node_modules/svelte/internal/index.mjs b/node_modules/svelte/internal/index.mjs
index c7f9b7c..22fa3bf 100644
--- a/node_modules/svelte/internal/index.mjs
+++ b/node_modules/svelte/internal/index.mjs
@@ -690,40 +690,9 @@ function is_crossorigin() {
     return crossorigin;
 }
 function add_resize_listener(node, fn) {
-    const computed_style = getComputedStyle(node);
-    if (computed_style.position === 'static') {
-        node.style.position = 'relative';
-    }
-    const iframe = element('iframe');
-    iframe.setAttribute('style', 'display: block; position: absolute; top: 0; left: 0; width: 100%; height: 100%; ' +
-        'overflow: hidden; border: 0; opacity: 0; pointer-events: none; z-index: -1;');
-    iframe.setAttribute('aria-hidden', 'true');
-    iframe.tabIndex = -1;
-    const crossorigin = is_crossorigin();
-    let unsubscribe;
-    if (crossorigin) {
-        iframe.src = "data:text/html,<script>onresize=function(){parent.postMessage(0,'*')}</script>";
-        unsubscribe = listen(window, 'message', (event) => {
-            if (event.source === iframe.contentWindow)
-                fn();
-        });
-    }
-    else {
-        iframe.src = 'about:blank';
-        iframe.onload = () => {
-            unsubscribe = listen(iframe.contentWindow, 'resize', fn);
-        };
-    }
-    append(node, iframe);
-    return () => {
-        if (crossorigin) {
-            unsubscribe();
-        }
-        else if (unsubscribe && iframe.contentWindow) {
-            unsubscribe();
-        }
-        detach(iframe);
-    };
+    const ro = new ResizeObserver(fn);
+    ro.observe(node);
+    return () => ro.disconnect();
 }
 function toggle_class(element, name, toggle) {
     element.classList[toggle ? 'add' : 'remove'](name);
