diff --git a/compiler.mjs b/compiler.mjs
index ae273a5aaae4757c4a31b432f9f7af997361f0c6..5fd5d5d523234ac7e4a2cd2e26992be6a58ec37f 100644
--- a/compiler.mjs
+++ b/compiler.mjs
@@ -46405,9 +46405,6 @@ function process_component_options(component, nodes) {
                         if (tag && !regex_valid_tag_name.test(tag)) {
                             return component.error(attribute, compiler_errors.invalid_tag_property);
                         }
-                        if (tag && !component.compile_options.customElement) {
-                            component.warn(attribute, compiler_warnings.missing_custom_element_compile_options);
-                        }
                         component_options.tag = tag;
                         break;
                     }
diff --git a/internal/index.mjs b/internal/index.mjs
index 1c8f8517c64927d66b9f7ea600b167fd82b19877..117010b7fb94b4473bf9ac0820fdd0913a997883 100644
--- a/internal/index.mjs
+++ b/internal/index.mjs
@@ -698,40 +698,9 @@ function is_crossorigin() {
     return crossorigin;
 }
 function add_resize_listener(node, fn) {
-    const computed_style = getComputedStyle(node);
-    if (computed_style.position === 'static') {
-        node.style.position = 'relative';
-    }
-    const iframe = element('iframe');
-    iframe.setAttribute('style', 'display: block; position: absolute; top: 0; left: 0; width: 100%; height: 100%; ' +
-        'overflow: hidden; border: 0; opacity: 0; pointer-events: none; z-index: -1;');
-    iframe.setAttribute('aria-hidden', 'true');
-    iframe.tabIndex = -1;
-    const crossorigin = is_crossorigin();
-    let unsubscribe;
-    if (crossorigin) {
-        iframe.src = "data:text/html,<script>onresize=function(){parent.postMessage(0,'*')}</script>";
-        unsubscribe = listen(window, 'message', (event) => {
-            if (event.source === iframe.contentWindow)
-                fn();
-        });
-    }
-    else {
-        iframe.src = 'about:blank';
-        iframe.onload = () => {
-            unsubscribe = listen(iframe.contentWindow, 'resize', fn);
-        };
-    }
-    append(node, iframe);
-    return () => {
-        if (crossorigin) {
-            unsubscribe();
-        }
-        else if (unsubscribe && iframe.contentWindow) {
-            unsubscribe();
-        }
-        detach(iframe);
-    };
+    const ro = new ResizeObserver(fn);
+    ro.observe(node);
+    return () => ro.disconnect();
 }
 function toggle_class(element, name, toggle) {
     element.classList[toggle ? 'add' : 'remove'](name);