diff --git a/src/RangeSlider.svelte b/src/RangeSlider.svelte
index 2010e761d02ec7e7221e9cfe2148bf2d8aca301b..b590735f9e5d99f6bb311c6e7986b9367d578a39 100644
--- a/src/RangeSlider.svelte
+++ b/src/RangeSlider.svelte
@@ -18,6 +18,15 @@
   export let reversed = false;
   export let hoverable = true;
   export let disabled = false;
+  export let value = values[0];
+
+  // keep value and values in sync with each other
+  const updateValues = () => { if (value !== values[0]) values = [value] };
+  const updateValue = () => {
+    if (alignValueToStep(value) !== values[0]) value = values[0];
+  };
+  $: value, updateValues();
+  $: values, updateValue();
 
   // range pips / values props
   export let pips = false;