diff --git a/node_modules/svelte/animate/index.mjs b/node_modules/svelte/animate/index.mjs
index 4d9ec6f..f93b20c 100644
--- a/node_modules/svelte/animate/index.mjs
+++ b/node_modules/svelte/animate/index.mjs
@@ -4,8 +4,8 @@ import { is_function } from '../internal/index.mjs';
 function flip(node, animation, params = {}) {
     const style = getComputedStyle(node);
     const transform = style.transform === 'none' ? '' : style.transform;
-    const scaleX = animation.from.width / node.clientWidth;
-    const scaleY = animation.from.height / node.clientHeight;
+    const scaleX = animation.from.width / node.offsetWidth;
+    const scaleY = animation.from.height / node.offsetHeight;
     const dx = (animation.from.left - animation.to.left) / scaleX;
     const dy = (animation.from.top - animation.to.top) / scaleY;
     const d = Math.sqrt(dx * dx + dy * dy);
