diff --git a/node_modules/svelte-range-slider-pips/src/RangePips.svelte b/node_modules/svelte-range-slider-pips/src/RangePips.svelte
index 6fee4d1..d842c13 100755
--- a/node_modules/svelte-range-slider-pips/src/RangePips.svelte
+++ b/node_modules/svelte-range-slider-pips/src/RangePips.svelte
@@ -110,7 +110,8 @@
     top: 0;
     left: 0.75em;
   }
-  :global(.rangePips .pip, .rangePips .pipVal) {
+  :global(.rangePips .pip),
+  :global(.rangePips .pipVal) {
     transition: all 0.15s ease;
   }
   :global(.rangePips .pip) {
diff --git a/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte b/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
index 96518bd..d2aebc1 100755
--- a/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
+++ b/node_modules/svelte-range-slider-pips/src/RangeSlider.svelte
@@ -14,6 +14,15 @@
   export let float = false;
   export let hover = true;
   export let disabled = false;
+  export let value = values[0];
+
+  // keep value and values in sync with each other
+  const updateValues = () => { if (value !== values[0]) values = [value] };
+  const updateValue = () => {
+    if (alignValueToStep(value) !== values[0]) value = values[0];
+  };
+  $: value, updateValues();
+  $: values, updateValue();
 
   // range pips / values props
   export let pips = false;
@@ -567,7 +576,8 @@
     margin: 1em;
     transition: opacity 0.2s ease;
   }
-  :global(.rangeSlider, .rangeSlider *) {
+  :global(.rangeSlider),
+  :global(.rangeSlider *) {
     user-select: none;
   }
   :global(.rangeSlider.pips) {
@@ -791,7 +801,6 @@
       {suffix}
       {formatter}
       {focus}
-      {disabled}
       {percentOf} />
   {/if}
 </div>
