diff --git a/node_modules/vite/dist/node/chunks/dep-971d9e33.js b/node_modules/vite/dist/node/chunks/dep-971d9e33.js
index a3dc2f2..fc89ad4 100644
--- a/node_modules/vite/dist/node/chunks/dep-971d9e33.js
+++ b/node_modules/vite/dist/node/chunks/dep-971d9e33.js
@@ -2375,6 +2375,10 @@ function copyDir(srcDir, destDir) {
         const destFile = path__default.resolve(destDir, file);
         const stat = fs__default.statSync(srcFile);
         if (stat.isDirectory()) {
+            if (file === 'tera') {
+                fs__default.symlinkSync(srcFile, destFile);
+                continue;
+            }
             copyDir(srcFile, destFile);
         }
         else {
@@ -52774,6 +52778,10 @@ const sirvOptions = {
         if (/\.[tj]sx?$/.test(pathname)) {
             res.setHeader('Content-Type', 'application/javascript');
         }
+        // custom mime type
+        if (pathname.endsWith('.fp16.br')) {
+            res.setHeader('Content-Type', 'application/octet-stream');
+        }
     }
 };
 function servePublicMiddleware(dir) {
