diff --git a/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js b/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js
index 4e00aba..07bb3e4 100644
--- a/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js
+++ b/node_modules/vite/dist/node/chunks/dep-85dbaaa7.js
@@ -4188,6 +4188,10 @@ function copyDir(srcDir, destDir) {
         const destFile = path__default.resolve(destDir, file);
         const stat = fs__default.statSync(srcFile);
         if (stat.isDirectory()) {
+            if (file === 'data') {
+                fs__default.symlinkSync(srcFile, destFile);
+                continue;
+            }
             copyDir(srcFile, destFile);
         }
         else {
