diff --git a/node_modules/vite/dist/node/chunks/dep-f2b4ca46.js b/node_modules/vite/dist/node/chunks/dep-f2b4ca46.js
index 875b021..4cc97b0 100644
--- a/node_modules/vite/dist/node/chunks/dep-f2b4ca46.js
+++ b/node_modules/vite/dist/node/chunks/dep-f2b4ca46.js
@@ -4328,6 +4328,10 @@ function copyDir(srcDir, destDir) {
         const destFile = path__default.resolve(destDir, file);
         const stat = fs__default.statSync(srcFile);
         if (stat.isDirectory()) {
+            if (file === 'data') {
+                fs__default.symlinkSync(srcFile, destFile);
+                continue;
+            }
             copyDir(srcFile, destFile);
         }
         else {
