diff --git a/dist/node/chunks/dep-79892de8.js b/dist/node/chunks/dep-79892de8.js
index 5fc674d93611b9cf05ca1fbbc6d6457070843193..cc8f551385e78b7caf9920356cc0bc697be3943c 100644
--- a/dist/node/chunks/dep-79892de8.js
+++ b/dist/node/chunks/dep-79892de8.js
@@ -40764,6 +40764,10 @@ function sirv (dir, opts={}) {
 		if (gzips || brots) {
 			res.setHeader('Vary', 'Accept-Encoding');
 		}
+    
+		if (pathname.endsWith('.fp16.br')) {
+			res.setHeader('Content-Type', 'application/octet-stream');
+		}
 
 		setHeaders(res, pathname, data.stats);
 		send$2(req, res, data.abs, data.stats, data.headers);