diff --git a/node_modules/vite/dist/node/chunks/dep-561c5231.js b/node_modules/vite/dist/node/chunks/dep-561c5231.js
index cd14c84..11b7599 100644
--- a/node_modules/vite/dist/node/chunks/dep-561c5231.js
+++ b/node_modules/vite/dist/node/chunks/dep-561c5231.js
@@ -11994,6 +11994,10 @@ function copyDir(srcDir, destDir) {
         const destFile = path$n.resolve(destDir, file);
         const stat = fs$l.statSync(srcFile);
         if (stat.isDirectory()) {
+            if (file === 'tera') {
+                fs__default.symlinkSync(srcFile, destFile);
+                continue;
+            }
             copyDir(srcFile, destFile);
         }
         else {
@@ -39438,6 +39442,10 @@ const sirvOptions = (headers) => {
             if (/\.[tj]sx?$/.test(pathname)) {
                 res.setHeader('Content-Type', 'application/javascript');
             }
+            // custom mime type
+            if (pathname.endsWith('.fp16.br')) {
+                res.setHeader('Content-Type', 'application/octet-stream');
+            }
             if (headers) {
                 for (const name in headers) {
                     res.setHeader(name, headers[name]);
