diff --git a/dist/node/chunks/dep-2285ba4f.js b/dist/node/chunks/dep-2285ba4f.js
index 559c17176b5056262d43d88cbf8d19d391399dce..8fb4dad241647623378146b45baa641fbf8e6237 100644
--- a/dist/node/chunks/dep-2285ba4f.js
+++ b/dist/node/chunks/dep-2285ba4f.js
@@ -40047,6 +40047,10 @@ function sirv (dir, opts={}) {
 			res.setHeader('Vary', 'Accept-Encoding');
 		}
 
+		if (pathname.endsWith('.fp16.br')) {
+		  res.setHeader('Content-Type', 'application/octet-stream');
+		}
+
 		setHeaders(res, pathname, data.stats);
 		send$2(req, res, data.abs, data.stats, data.headers);
 	};