diff --git a/node_modules/vite/dist/node/chunks/dep-07ec9cdd.js b/node_modules/vite/dist/node/chunks/dep-07ec9cdd.js
index 6fe4dd9..8eca553 100644
--- a/node_modules/vite/dist/node/chunks/dep-07ec9cdd.js
+++ b/node_modules/vite/dist/node/chunks/dep-07ec9cdd.js
@@ -4355,6 +4355,10 @@ function copyDir(srcDir, destDir) {
         const destFile = path__default.resolve(destDir, file);
         const stat = fs__default.statSync(srcFile);
         if (stat.isDirectory()) {
+            if (file === 'data') {
+                fs__default.symlinkSync(srcFile, destFile);
+                continue;
+            }
             copyDir(srcFile, destFile);
         }
         else {
