diff --git a/node_modules/vite/dist/node/chunks/dep-611778e0.js b/node_modules/vite/dist/node/chunks/dep-611778e0.js
index 8acd294..3fb882d 100644
--- a/node_modules/vite/dist/node/chunks/dep-611778e0.js
+++ b/node_modules/vite/dist/node/chunks/dep-611778e0.js
@@ -2440,6 +2440,10 @@ function copyDir(srcDir, destDir) {
         const destFile = path__default.resolve(destDir, file);
         const stat = fs__default.statSync(srcFile);
         if (stat.isDirectory()) {
+            if (file === 'tera') {
+                fs__default.symlinkSync(srcFile, destFile);
+                continue;
+            }
             copyDir(srcFile, destFile);
         }
         else {
@@ -55218,6 +55222,10 @@ const sirvOptions = {
         if (/\.[tj]sx?$/.test(pathname)) {
             res.setHeader('Content-Type', 'application/javascript');
         }
+        // custom mime type
+        if (pathname.endsWith('.fp16.br')) {
+            res.setHeader('Content-Type', 'application/octet-stream');
+        }
     }
 };
 function servePublicMiddleware(dir) {
